cgroup/util.go

```diff
@@ -174,6 +178,10 @@ func SubsystemMountpoints(rootfsMountpoint string, subsystems map[string]struct{
 			continue
 		}
 
+		if !strings.HasPrefix(mount.mountpoint, rootfsMountpoint) {
```

Comment:
A code comment would be helpful that this helps to filter out cgroups mounted under a different directory.